src/App.tsx

```diff
@@ -1,61 +1,41 @@
 import './App.scss';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { getUserById } from './services/user';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm';
+
+export const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+function getNewTodotId(todos: Todo[]) {
+  const maxId = Math.max(...todos.map(todo => todo.id));
+
+  return maxId + 1;
+}
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(initialTodos);
+  const addTodo = ({ id, ...data }: Todo) => {
+    const newTodo = {
+      id: getNewTodotId(todos),
+      ...data,
+    };
+
+    setTodos(currentTodo => [...currentTodo, newTodo]);
```

Comment:
Typo in variable name 'currentTodo'. Should be 'currentTodos' as it refers to multiple todos, not a single one.